paparazzi-preview-processor/src/test/java/app/cash/paparazzi/preview/processor/PreviewProcessorProviderTest.kt

```diff
@@ -289,5 +292,19 @@ class PreviewProcessorProviderTest {
         annotation class Paparazzi
         """.trimIndent()
       )
+
+    private val PAPARAZZI_PREVIEW_DATA_SOURCE = SourceFile.kotlin(
+      "PaparazziPreviewData.kt",
+      """
+        package app.cash.paparazzi.annotations
+
+        import androidx.compose.runtime.Composable
+
+        data class PaparazziPreviewData(
+          val snapshotName: String,
+          val composable: @Composable () -> Unit
+        )
+      """.trimIndent()
+    )
```

Comment:
accidental inclusion from another PR?

Reply:
nope, actually updating the testing library failed some tests as the generated code was referencing this class that was never defined before.